content/root/best-practice-guide.textile

```diff
@@ -19,28 +19,57 @@ h2(#ably-101). Ably 101
 
-h3(#presence). Presence
+**Important:** Do not use basic authentication on the client-side, as it exposes your Ably API key.
 
-With Ably's "Presence":/realtime/presence feature, your clients can attach to a presence channel and announce to the rest of the users that they are present on a channel. This is very useful, for example, in chat apps to indicate a member is online, or with IoT devices to indicate a device is online and connected. Clients can subscribe to realtime presence changes on channels, such as members entering or leaving, and respond to these events accordingly.
+Token authentication allows you to connect your auth server to Ably, using the Ably API key. Your clients can then receive token requests from the auth server, and then authenticate directly with Ably using a token request. These short-lived tokens provide a secure way to pass information over the internet and make sure that the Ably API key is never exposed to your client-side applications. However, the secure auth server can use the API key directly to authenticate with Ably using basic authentication, as on the secure server the API key will not be accessible to anyone other than autheticated personnel, such as systems administrators.
+
+**Important:** Do not use token authetication on the server, as this will result in unnecessary overhead - the server will periodically require token requests from Ably to authenticate. Simply use basic authentication server-side.
+
+If you prefer to use a single auth strategy across all third-party services, you can use "JWT Authentication":/core-features/authentication#ably-jwt-process as part of Token authentication.
+
+h3(#using-api-key-client-side). Using the API key on the client side
+
+Ably's "Basic authentication":/core-features/authentication#basic-authentication method is the easiest way to authenticate your clients on the Ably platform, but, this will directly expose the API key in the source code. Hence, when using it on the client side, any entity that interacts with the client could view your API key and potentially misuse it. So, if you can run an auth server, we recommend using "Token authentication":/core-features/authentication#token-authentication instead. With this method, your auth server is expected to use the API key to raise a token request from Ably, and the token can then be used by your end clients in the authentication step, eliminating the chance of misusing an exposed API key. "Read more":/core-features/authentication#selecting-auth about selecting an authentication mechanism.
```

Comment:
Just as a general note, we need to decide whether these auth methods deserve capitals - I would suggest not and that we reserved title case for product names.

Reply:
Yes, should be lowercase. I've fixed for these files, but suspect the issue may be more widespread.